Backend/Backend/Controllers/EpisodeController.cs

```diff
@@ -84,7 +84,7 @@ public async Task<IActionResult> AddEpisode([FromForm] CreateEpisodeRequest crea
                             try
                             {
                                 Episode? episode = await _episodeService!.AddEpisode(createEpisodRequest,podcast,HttpContext);
```

Comment:
you should probably add a validation afterwards if you are going to return an object, or just not use it